### ISSUE TITLE
Windows Defender flags stripe.exe CLI binary as threat

### ISSUE THREAD

**Issue by drchilds** [stripe/stripe-cli#627]
I just downloaded the latest release (1.5.12 Windows X86-64) of the Stripe CLI from the releases section here, and Windows Defender has flagged the executable therein as containing the threat PUA:Win32/Pearfoos.A!ml.

**Comment by tomer-stripe** [comment]
Hey @drchilds is this still happening? For some reason we've had some flakiness with Windows/Virus Total occasionally flipping back and forth on the file.

**Comment by drchilds** [comment]
Yes. Happened with the version mentioned on the day I reported it.

**Comment by jonjibbed** [comment]
Just had same issue. My corporate IT is quite tight, in case that helps.

**Comment by armanyfelix** [comment]
I'm having the same problem with the version 1.13.6
